api-specifications/properties.json

```diff
@@ -351,6 +351,11 @@
             "optional": true,
             "type": "string"
           },
+          "chartId": {
```

Comment:
This is bit strange, but I'm pretty sure we actually update this file when we release, not when we make changes to object-properties. @niekvanstaveren do you remember if this is true? In that case, this should be reverted

Reply:
This is an API specification error I got in CI so here is the result of running `yarn spec`

Reply:
No the workflow will fail when the api specification is not updated properly. The version of this needs to be updated when released, but the specifications should already be updated. 
This PR should however not be a refactor. You cannot do a refactor and change the API

Reply:
@niekvanstaveren ok didn't know that, should I change it to `chore`?

Reply:
API changes should probably always be `feat`. In very rare occasions I guess it could be a `fix`, but that would only be when you are fixing something in the API